int32 faceCount
int32 frameWidth
int32 frameHeight
face[] faces
